tests/test_system.py

```diff
@@ -2,535 +2,64 @@
-    try:
-        imported_module = importlib.import_module(module_name)
-    except ImportError:
-        assert False, "Failed to import module: {0}".format(module_name)
-
-    assert imported_module is not None, "Module {0} is None after import.".format(
-        module_name
-    )
-
-    _cleanup()
-
-
-def test_security():
-    module_name = "system.security"
-
-    try:
-        imported_module = importlib.import_module(module_name)
-    except ImportError:
-        assert False, "Failed to import module: {0}".format(module_name)
-
-    assert imported_module is not None, "Module {0} is None after import.".format(
-        module_name
-    )
-
-    _cleanup()
-
-
-def test_serial():
-    module_name = "system.serial"
-
-    try:
-        imported_module = importlib.import_module(module_name)
-    except ImportError:
-        assert False, "Failed to import module: {0}".format(module_name)
-
-    assert imported_module is not None, "Module {0} is None after import.".format(
-        module_name
-    )
-
-    _cleanup()
-
-
-def test_sfc():
-    module_name = "system.sfc"
-
-    try:
-        imported_module = importlib.import_module(module_name)
-    except ImportError:
-        assert False, "Failed to import module: {0}".format(module_name)
-
-    assert imported_module is not None, "Module {0} is None after import.".format(
-        module_name
-    )
-
-    _cleanup()
-
-
-def test_tag():
-    module_name = "system.tag"
-
-    try:
-        imported_module = importlib.import_module(module_name)
-    except ImportError:
-        assert False, "Failed to import module: {0}".format(module_name)
-
-    assert imported_module is not None, "Module {0} is None after import.".format(
-        module_name
-    )
-
-    _cleanup()
-
-
-def test_twilio():
-    module_name = "system.twilio"
-
-    try:
-        imported_module = importlib.import_module(module_name)
-    except ImportError:
-        assert False, "Failed to import module: {0}".format(module_name)
-
-    assert imported_module is not None, "Module {0} is None after import.".format(
-        module_name
-    )
-
-    _cleanup()
-
-
-def test_user():
-    module_name = "system.user"
-
-    try:
-        imported_module = importlib.import_module(module_name)
-    except ImportError:
-        assert False, "Failed to import module: {0}".format(module_name)
-
-    assert imported_module is not None, "Module {0} is None after import.".format(
-        module_name
-    )
-
-    _cleanup()
-
-
-def test_util():
-    module_name = "system.util"
-
-    try:
-        imported_module = importlib.import_module(module_name)
-    except ImportError:
-        assert False, "Failed to import module: {0}".format(module_name)
-
-    assert imported_module is not None, "Module {0} is None after import.".format(
-        module_name
-    )
-
-    _cleanup()
-
-
-def test_vision():
-    module_name = "system.vision"
-
+@pytest.mark.parametrize("test_module", _SYSTEM_MODULES)
+def test_system(test_module):
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (testing):** Add assertions for expected functionality within imported modules.

While the current test ensures that modules can be imported without errors, it might be valuable to also assert some expected functionality or state within each imported module. This could help catch more subtle integration issues that aren't exposed through import errors alone.